packages/api/amplify_api/example/integration_test/util.dart

```diff
@@ -89,10 +87,36 @@ Future<void> configureAmplify() async {
         options: APIPluginOptions(modelProvider: ModelProvider.instance),
       ),
     ]);
-    await Amplify.configure(amplifyconfig);
+    await Amplify.configure(gen1.amplifyConfig);
+  }
+}
+
+Future<void> configureAmplifyGen2() async {
```

Comment:
Q: Rather than having two functions, can you have one function with a boolean argument? I think it would make a lot of the tests a bit cleaner.

```dart
Future<void> configureAmplify({bool useGen1Config = false}) async {
  if (!Amplify.isConfigured) {
    await Amplify.addPlugins([
      AmplifyAuthCognito(
        secureStorageFactory: AmplifySecureStorage.factoryFrom(
          macOSOptions: MacOSSecureStorageOptions(useDataProtection: false),
        ),
      ),
      AmplifyAPI(
        options: APIPluginOptions(modelProvider: ModelProvider.instance),
      ),
    ]);
    if (useGen1Config) {
      await Amplify.configure(amplifyconfig);
    } else {
      await Amplify.configure(amplify_outputs.amplifyConfig);
    }
  }
}
```